packages/elemental/elemental/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,46 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,28 +1,37 @@
+ image:
+   empty: rancher/pause:3.1
+-  # elemental-operator image is also built on:
+-  #   registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-operator
+-  repository: "quay.io/costoolkit/elemental-operator"
+-  tag: v1.2.5
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-operator/5.3"
++  tag: "1.2.5"
+   imagePullPolicy: IfNotPresent
++
+ seedImage:
+-  repository: "quay.io/costoolkit/elemental-seedimage"
+-  tag: v1.2.5
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/seedimage-builder/5.3"
++  tag: "1.2.5"
+   imagePullPolicy: IfNotPresent
++
++channel:
++  repository: "registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-teal-channel/5.3" 
```

Comment:
This should be `rancher` not `registry.opensuse.org`. You can add your docker image to https://github.com/rancher/image-mirror before submitting any new chart.